mesonbuild/compilers/cpp.py

```diff
@@ -349,9 +349,10 @@ class AppleClangCPPCompiler(AppleCompilerMixin, AppleCPPStdsMixin, ClangCPPCompi
     def is_libcpp_enable_assertions_deprecated(self) -> bool:
```

Comment:
There is likely a word missing in the commit message subject line, after "old".

Also, the tags in the commit message likely need to be "Fixes" and not "Bug" to have GitHub close the issues automatically when this is merged.

Reply:
"Closes" works too, which in context...

("Bug" / "Closes" is part of Gentoo's automation, depending on whether you want a reference or a close annotation, and is specifically intended to be cross-compatible with bugzilla and Github.)

Reply:
> I was using `Bug` partly out of reflex because the user hadn't confirmed at the time, will change it for the still-open bug (would prefer to keep Bug for the other, older one though).

Yes, quite. For the older one we don't in fact want it to look like it was closed by this patch. :)